.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 Make sure to follow and check these boxes before submitting an issue! Thank you.
 
-- [ ] Supply `iocage --version`
+- [ ] Supply `ioc --version`
```

Comment:
The ioc module now provides a `VERSION` property that prints the lib version currently used. The CLI tool should embed it, so that version differences can be detected.

```python
import ioc
print(ioc.VERSION)
```